server/routes/user.js

```diff
@@ -0,0 +1,158 @@
+import { Router } from 'express'
+import ensureAuthenticated from '../utils/passport/ensure-authenticated'
+
+const router = Router()
+
+router
+  .route(`/api/v1/users/:uid/site-subjects`)
```

Comment:
this should use a route constant.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/bf0423c6c7f4d2b24d8c720cb39b4accdaccbbc7